staker-rewards/lib/numbers.js

```diff
@@ -0,0 +1,6 @@
+import BigNumber from "bignumber.js"
+
+export const decimalPlaces = 2
+
+export const shorten18Decimals = (value) =>
+  new BigNumber(value).dividedBy(new BigNumber(1e18)).toFixed(decimalPlaces)
```

Comment:
When we are displaying a number in 1e18 format, we should not display any decimals as we do not want to go beyond 1e18 precision.

Please see https://github.com/keep-network/keep-ecdsa/pull/632/commits/e382851b99dc1efa9ec413337ffff1b7083bf48d and https://github.com/keep-network/keep-ecdsa/pull/632/commits/1441b5f6d219071a8f624d190e1792e55742870e.

Reply:
I'm not sure I follow. It's just a function extracted from `rewards.js`. It's in this form in master:
https://github.com/keep-network/keep-ecdsa/blob/985d0749885087894a385562f30df51c9da7da1f/staker-rewards/rewards.js#L257-L258

Reply:
This function will be used to convert wei to ether so decimals are very much required.

Reply:
I think I missed how this function is used. I will resolve the comment for now and go through the code again.